components/src/Button/Button.js

```diff
@@ -95,9 +93,17 @@ const Button = styled(BaseButton)`
     &:hover {
       background-color: ${props => (props.disabled ? null : props.theme.colors.lightBlue)};
     }
-    &:focus {outline: none;}
-    &:active {transform: scale(0.98); transition: .2s ease-in;}
-    &:disabled {opacity: .5;}
+    &:focus {
```

Comment:
this could all be combined with just `&:hover, &:focus {}`

Reply:
After looking at it more - what do you think of using the hover styling as the focus? Mainly because it kinda breaks the illusion that the component is a physical button. ie. when you hover -> click the button they styling appears the same as when you hovered almost making it feel like that button hasn't "released" and is still pressed into the screen. Very subjective of course but let me know your thoughts.